### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by JustBlan1984** [baldsealion/BSL-Tools-ROG-ALLY-Edition#2]
Missing “j” error when trying to run this program

**Comment by baldsealion** [comment]
Hi ! A new release was just published that fixes this issue. Please download, extract and overwrite the old files! Be sure to read the release notes!

**Comment by JustBlan1984** [comment]
I just tried 1.11 and when I try to start BSL Tools I still get the Cannot See TDP missing RyzenAdj error.Sent from my iPhoneOn Nov 16, 2023, at 12:18 AM, baldsealion ***@***.***> wrote:﻿
Closed #2 as completed.

**Comment by baldsealion** [comment]
How are you extracting this?
Maybe try the zip file instead in releases?

you should have both a “config” and a “ryzenadj” folder inside of the bsl-tools folder. That error will come up because you are actually missing those files.

**Comment by JustBlan1984** [comment]
Yes they are both in the same folder. I extracted with 7zip to a new folder I renamed to BSL Tools on my desktop. Sent from my iPhoneOn Nov 16, 2023, at 7:37 AM, baldsealion ***@***.***> wrote:﻿
How are you extracting this?
Maybe try the zip file instead in releases?
you should have both a “config” and a “ryzenadj” folder inside of the bsl-tools folder. That error will come up because you are actually missing those files.

**Comment by baldsealion** [comment]
When is this message come up? On launch or when you are trying to use TDP hotkeys?

I’ve had two others test the new release so far with no issues. 

To make things simpler I have changed the release to be just a zip file now. Please download that and extract the entire folder and all sub folders.

**Comment by JustBlan1984** [comment]
On lunchSent from my iPhoneOn Nov 16, 2023, at 8:09 AM, baldsealion ***@***.***> wrote:﻿
When is this message come up? On launch or when you are trying to use TDP hotkeys?
I’ve had two others test the new release so far with no issues.
To make things simpler I have changed the release to be just a zip file now. Please download that and extract the entire folder and all sub folders.

**Comment by JustBlan1984** [comment]
I literally just did whole fresh install and set my whole pc back up and downloaded BSL Tools 1.11 extracted to desktop folder named BSL Tools tried to launch it and same error.Sent from my iPhoneOn Nov 16, 2023, at 8:09 AM, baldsealion ***@***.***> wrote:﻿
When is this message come up? On launch or when you are trying to use TDP hotkeys?
I’ve had two others test the new release so far with no issues.
To make things simpler I have changed the release to be just a zip file now. Please download that and extract the entire folder and all sub folders.

**Comment by baldsealion** [comment]
@JustBlan1984 I have no idea why you are experiencing this issue. Do you have some kind of antivirus that could be removing RyzenADJ?

I just extracted the zip to my desktop under the folder "C:\Users\ALLY\Desktop\BSL-Tools-ROG-ALLY-Edition-1.11"

There is no issues, no error messages. The only time RyzenADJ error message should come up is if the Ryzenadj.exe file is missing in the Ryzenadj folder. That's it.

![image](https://github.com/baldsealion/BSL-Tools-ROG-ALLY-Edition/assets/93551277/3e8f2142-3099-468a-9d77-6e1f20e54952)

**Comment by JustBlan1984** [comment]
The RyzenAdj.exe is still there in the folderSent from my iPhoneOn Nov 16, 2023, at 3:35 PM, baldsealion ***@***.***> wrote:﻿
@JustBlan1984 I have no idea why you are experiencing this issue. Do you have some kind of antivirus that could be removing RyzenADJ?
I just extracted the zip to my desktop under the folder "C:\Users\ALLY\Desktop\BSL-Tools-ROG-ALLY-Edition-1.11"
There is no issues, no error messages. The only time RyzenADJ error message should come up is if the Ryzenadj.exe file is missing in the Ryzenadj folder. That's it.

**Comment by wetini** [comment]
@JustBlan1984 I had the same issue. I originally extracted to a folder called "BSL Tools", after i removed the space and named it "BSL-Tools" it worked.
